weave/trace/op_type.py

```diff
@@ -267,6 +269,16 @@ def get_source_or_fallback(fn: typing.Callable, *, warnings: list[str]) -> str:
     if isinstance(fn, Op):
         fn = fn.resolve_fn
 
+    if not settings.should_capture_code():
+        # This digest is kept for op versioning purposes
+        digest = str_digest(inspect.getsource(fn))
+        return textwrap.dedent(
+            f"""
+            def func(*args, **kwargs):
+                ...  # Code-capture disabled for this op (digest: {digest})
```

Comment:
nit: "Code-capture was disabled while saving this op" (code capture is not a setting on the op itself)